lib/worker.js

```diff
@@ -72,14 +72,45 @@ async function maybeRetrievePayload(job) {
 
 async function invokeLabeling(job) {
   const { resource_id, payload, fhir_base } = job;
-  const slsResponse = await superagent
-    .post(SLS_ENDPOINT)
-    .send(bundle(payload, fhir_base, resource_id));
+  const bundled = bundle(payload, fhir_base, resource_id);
 
-  // return superagent.post(job.fhir_base).send(slsResponse.body);
-  return hapiBugWorkaround(job.fhir_base, slsResponse.body);
+  const slsResponse = await superagent.post(SLS_ENDPOINT).send(bundled);
+
+  if (process.env.FHIR_SERVER_TYPE === "hapi") {
+    return hapiBugWorkaround(job.fhir_base, slsResponse.body);
+  } else if (process.env.FHIR_SERVER_TYPE === "medplum") {
+    return medplumBugWorkaround(job.fhir_base, slsResponse.body);
+  } else {
+    return superagent.post(job.fhir_base).send(slsResponse.body);
+  }
 }
 
+const medplumBugWorkaround = (fhir_base, slsResponseBody) => {
```

Comment:
I think it will be better if this is not called 'medplumBug' workaround and instead we call it 'patch approach' and use this approach in the deployed version in the sandbox.